Wobble/Graphics/Drawable.cs

```diff
@@ -615,6 +623,55 @@ public void ClearAnimations()
         /// <returns></returns>
         public bool IsHovered() => GraphicsHelper.RectangleContains(ScreenRectangle, MouseManager.CurrentState.Position);
 
+        /// <summary>
+        ///     Removes all previously scheduled updates, and schedules a new one to run in the next frame
+        /// </summary>
+        /// <param name="action"></param>
+        public void ScheduleUpdate(Action action)
```

Comment:
Not sure this is a useful method if we consider that different parts of the system will schedule their own updates. So you probably don't want some random thing just clearing all of them?